lib/our_fitness_pal_api/accounts.ex

```diff
@@ -18,7 +18,9 @@ defmodule OurFitnessPalApi.Accounts do
 
   """
   def list_users do
-    Repo.all(User)
+    query = from user in User,
+      select: {user.id, user.email, user.inserted_at, user.updated_at}
+    Repo.all(query)
```

Comment:
This looks pretty good, the only minor I would change is to pipe the query into the Repo call:

```elixir
from(user in User, select: {user.id, user.email, user.inserted_at, user.updated_at})
|> Repo.all
```